src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/SearchController.php

```diff
@@ -65,4 +69,32 @@ public function indexAction(Request $request, AliasFilters $filters): Response
             ],
         ]);
     }
+
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function searchAliasesForAssociationAction(Request $request): JsonResponse
+    {
+        try {
+            /** @var AliasForAssociation[] $searchTerms */
+            $searchTerms = $this->getQueryBus()->handle(new SearchAliasesForAssociation(
+                $request->get('query', ''),
+                (int) $request->get('limit', 20)
+            ));
+        } catch (AliasConstraintException $e) {
+            return $this->json([
+                'message' => $e->getMessage(),
```

Comment:
hmm $e->getMessage is not suitable for end use. 
You should provide a translated error message based on thrown exception. (Like in all other actions)